src/viur/datastore/cache.py

```diff
@@ -132,3 +135,40 @@ def check_for_memcache() -> bool:
 		logging.warning(f"""conf["memcache_client"] is 'None'. It can not be used.""")
 		return False
 	return True
+
+
+class LocalMemcache:
+	def __init__(self):
+		self._data = {}
+
+	def get_multi(self, keys: List[str], namespace: str = MEMCACHE_NAMESPACE):
+		if not self._data.get(namespace):
+			self._data[namespace] = {}
+		res = {}
+		for key in keys:
+			data = self._data[namespace].get(key)
+			if data is not None:
```

Comment:
As @ArneGudermann would suggest: why not use the walrus operator? :)

```suggestion
			if (data := self._data[namespace].get(key)) is not None:
```